apps/admin-be/src/controllers/authController.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response } from "express";
+
+const Login = async (req: Request, res: Response): Promise<void> => {
+  if (!req.body.firstName) {
+    res.status(400).json("no first name");
+  }
+  res.sendStatus(201);
+};
+
+const AuthController = {
+  Login,
+};
+
+export { AuthController };
```

Comment:
Can we add a comment here that states that this auth is purely boilerplate. 

Later in another PR we will import AWS Cognito.